capsul/process/xml.py

```diff
@@ -21,6 +21,7 @@
 
 _known_values = {
     'Undefined': Undefined,
+    'None': Undefined
```

Comment:
I think `"None"` should set the value `None` or be forbidden in XML. Asking for `None` and having an `Undefined` value seems weird.